packages/react-router/__tests__/useResolvedPath-test.tsx

```diff
@@ -32,6 +32,33 @@ describe("useResolvedPath", () => {
     });
   });
 
+  it("partial path object resolves to Path object", () => {
+    let path!: Path;
+    function Home() {
+      path = useResolvedPath({
+        pathname: "/home",
+        search: new URLSearchParams({ user: "mj" }).toString(),
```

Comment:
You could also just do:

```ts
search: "?user=mj"
```

which I think is a little more direct. Unless ofc you're trying to test the behavior of URLSearchParams#toString, which it doesn't seem like.

Reply:
I was writing the test as I figured a user might write their code, as they did in the root issue, but that would translate to `search: "user=mj"` I believe, yeah?